charts/ocis/values.yaml

```diff
@@ -4612,14 +4612,14 @@ services:
         # type: string
         # required: true
         # @schema
-        # -- Setting that makes the command delete all trashed personal files older than the value. The value is a number and a unit "d", "h", "m", "s".
-        personalDeleteBefore: 30d
+        # -- Setting that makes the command delete all trashed personal files older than the value. The value is a number and a unit "h", "m", "s".
```

Comment:
can you imagine an easy pattern to match valid values? see https://github.com/dadav/helm-schema?tab=readme-ov-file#pattern

Reply:
> Probably the most problematic part would be composite durations like 1h5m3s, if they should be supported at all!?

Infinite Scale supports mixed units, see here https://doc.owncloud.com/ocis/next/deployment/services/envvar-types-description.html

```
Durations can be combined by multiple numbers and their identifiers
OCIS_CACHE_TTL=336h0m0s
```

Whether or not we want to support them in the helmchart is a different topic.

Reply:
I would not support them. All mixed values can still be written by using the lowest entity and multiplying the other in there, e.g. `336h30m10s` => `1211410s`

Reply:
> can you imagine an easy pattern to match valid values? see https://github.com/dadav/helm-schema?tab=readme-ov-file#pattern and https://json-schema.org/understanding-json-schema/reference/regular_expressions

pattern should then be `^[0-9]*[hms]$` depending on the RegEx dialect used.

Reply:
Added and tested in code.